src/middlewares/isCommentBlocked.js

```diff
@@ -0,0 +1,68 @@
+import models from '../api/models';
```

Comment:
this also could be used with the same middleware. 
I really have an Idea

Reply:
@nkpremices, we have a simple way to refactor this after @MemunaHaruna merge